src/Sarif/Writers/SarifLogger.cs

```diff
@@ -150,12 +142,11 @@ private static string Redact(string text, IEnumerable<string> tokensToRedact)
 
             _run = CreateRun(
                 analysisTargets,
-                computeTargetsHash,
-                logEnvironment,
+                ComputeFileHashes,
```

Comment:
>ComputeFileHashes [](start = 16, length = 17)

Why pass these as arguments if they are available as properties?

Reply:
calling a static method

Reply:
calling static method

---
In reply to: [135841212](https://github.com/Microsoft/sarif-sdk/pull/609#discussion_r135841212) [](ancestors = 135841212)